src/mount/writedata.cc

```diff
@@ -1037,3 +1040,1269 @@ int write_data_end(void* vid) {
 	write_data_lcnt_decrease(id, lock);
 	return status;
 }
+
+}  // namespace OldWriteAlgorithm
+
+namespace NewWriteAlgorithm {
```

Comment:
Consider ChunkBasedWriteAlgorithm and INodeBasedWriteAlgorithm. The word new gets old very frequently.

Reply:
Applied.